src/States/PaymentRun/PaymentRunState.php

```diff
@@ -15,4 +15,9 @@ public static function getFrontendFormatter(...$args): string|array
                 ->map(fn ($key) => (new $key(''))->color()),
         ];
     }
+
+    public static function config(): \Spatie\ModelStates\StateConfig
```

Comment:
swap function ordering with getFrontendFormatter() to keep it in line with order states